### ISSUE TITLE
Bug: Launching code from the wsl command line shouldn't break tmux.

### ISSUE THREAD

**Issue by kumarharsh** [microsoft/terminal#1599]
# Environment

```none
Windows build number: [run "ver" at a command prompt]: 10.0.18362.175
Windows Terminal version (if applicable): 0.2.1715.0

Any other software?
```

# Steps to reproduce

I'm running `tmux` in `wsl`. When I open vscode in any pane by writing `code .` or `code-insiders .`, the whole tmux display gets mutilated. Starting typing something in one pane starts changing other side panes.

# Expected behavior

Launching code from the wsl command line shouldn't break tmux.

# Actual behavior

The panes get broken - maybe it's a conpty bug, because I also get similar issues when running Alacritty using conpty. I can also intermittently see the cursor jumping between different panes for a brief moment (100ms or so)...

**Comment by JushBJJ** [comment]
Report this in the wsl git repo.

**Comment by kumarharsh** [comment]
It's a conpty issue IMO

**Comment by DHowett-MSFT** [comment]
Does this reproduce outside of Windows Terminal?

**Comment by DHowett-MSFT** [comment]
Sorry, accidental close.

**Comment by kumarharsh** [comment]
Yes, with Alacritty - that's why I wrote this might be a conpty thing

**Comment by DHowett-MSFT** [comment]
Okay how about *stock Windows console*

**Comment by kumarharsh** [comment]
No, I don't believe it happens with that.
